Userland/Games/Solitaire/Game.cpp

```diff
@@ -19,24 +19,30 @@ namespace Solitaire {
 static constexpr uint8_t new_game_animation_delay = 2;
 static constexpr int s_timer_interval_ms = 1000 / 60;
 
-Game::Game()
+ErrorOr<NonnullRefPtr<Game>> Game::try_create()
```

Comment:
`try_create()` is reserved for when there already is an infallible `create()`, not for arbitrary fallible factory functions.

Reply:
(this applies to every `try_foo()`, not just `create()`)

Reply:
`C_OBJECT()` creates `construct()` and `try_create()`. https://github.com/SerenityOS/serenity/blob/master/Userland/Libraries/LibCore/Object.h#L74-L83

Also, the fallible GML loading I wrote (which got merged a little while ago) relies on `try_create()` being the factory name, because it exists for all Widgets already, and then to add additional fallible method calls you do what I did here.

There's probably a better way but I'm not sure what it is.

Reply:
eh, fine, needs a cleanup some day